packages/renderer/src/lib/pod/PodsList.svelte

```diff
@@ -31,23 +31,32 @@ import PodColumnStatus from './PodColumnStatus.svelte';
 import PodEmptyScreen from './PodEmptyScreen.svelte';
 import type { PodInfoUI } from './PodInfoUI';
 
-export let searchTerm = '';
-$: searchPattern.set(searchTerm);
+interface Props {
+  searchTerm?: string;
+}
+
+let { searchTerm = $bindable('') }: Props = $props();
+
+$effect(() => {
+  searchPattern.set(searchTerm);
+});
```

Comment:
remark (non-blocking): this pattern is weird, not blocking as was already the case in the non-rune mode, but this is looking odd